src/services/response.service.js

```diff
@@ -38,7 +38,7 @@ export default class ResponseService {
      */
   static send(res) {
     const result = {
-      status: this.type,
+      status: this.statusCode,
```

Comment:
Here it was bringing an incorrect response(status) instead of a number. that's why I edited this file. We didn't notice earlier